recipes/categorical_anomaly.py

```diff
@@ -163,6 +164,21 @@ def raster_to_vector(self, geotif_path=None):
             return self.df
 
 
+    def normalize(self, field_1 = None,):
```

Comment:
Ah because initially I had the option to normalize by two fields - ex. population, then area to give a clicks per person per area sort of thing, but then Steven kind of mentioned that might not be a good idea, so I removed the option for the second level of normalization, but kept the variable name. Haha I can change it back

Reply:
i see. reasonable, as long as it's documented so (especially for ppl who tend to always provide both `key=value` in function/method calls)